CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Master
 
+### Enhancements
+
+- added an optional second parameter to the `include` tag for passing a sub context to the included file
```

Comment:
Maybe we should directly start applying the new changelog format? End with a `.  ` (2 spaces), and the author + issue link. See https://github.com/SwiftGen/SwiftGen/blob/master/CHANGELOG.md for examples.

Nitpick: start sentences with an uppercase.

Reply:
Cool, done